src/Paket.Core/Dependencies/PackageResolver.fs

```diff
@@ -1264,10 +1264,7 @@ let Resolve (getVersionsRaw : PackageVersionsFunc, getPreferredVersionsRaw : Pre
                             UnlistedSearch = true
                         }
                     else
-                        { flags with 
-                            Ready = true 
-                            UnlistedSearch = true 
-                        }
+                        flags
```

Comment:
/cc @cloudRoutine @matthid 

I'm scared!

Reply:
You even added a test-case for this scenario. What could go wrong if everything is green ;)

Reply:
Do we have test-cases for "unlisted" scenarios?

Reply:
I think there are integration tests.